index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube Player</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <!--youtube sample webpage-->
+    <header>
+        <img class="logo-nav-bar" src="./images/ytlogo.jpg" alt="">
+        <nav class="navbar">
+            <div class="nav-bar-cascade">
+                <ul>
+                    <il><a href="#">Home</a></il>
+                    <il><a href="#">Shorts</a></il>
+                    <il><a href="#">Suscriptions</a></il>
+                    <il><a href="#">Library</a></il>
+                </ul>
+            </div> 
+            <div class="nav-bar-search-bar">
+                <input type="text" placeholder="Search">
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="player-container">
```

Comment:
- [ ] Good job with the use of inline comments 👏 . However, You're meant to check the code to look for errors and bugs and make nice comments containing change suggestions for each. 
- [ ] Examples of issues that you  may check when performing the review:
- Lack of [semantic HTML tags use](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Not applying [HTML & CSS best practices correclty](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- Bugs you missed the first time the project was built.
- etc

- [ ] When making your suggestions from bugs or best practices missed, kindly include an explanation, a "why", for each requested change. For example:

* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*